manage-data/data-store/templates.md

```diff
@@ -49,15 +102,13 @@ If you use {{fleet}} or {{agent}}, assign your index templates a priority lower
 * To avoid naming collisions with built-in and Fleet-managed index templates, avoid using `@` as part of the name of your own index templates.
 * Beginning in {{stack}} version 9.1, {{fleet}} uses indices named `fleet-synced-integrations*` for a feature. Avoid using this name to avoid collisions with built-in indices.
 
-::::
-
+## Component templates [component-templates]
 
+A **component template** is a reusable building block, containing mappings, settings, and aliases, used to construct an index template. Component templates are used only to construct index templates and are not applied to indices directly.
```

Comment:
Used to construct index templates sound a bit weird when we want to mean that index templates include component templates to reuse their contents.

```suggestion
A **component template** is a reusable building block that defines mappings, settings, and aliases. Component templates are not applied directly to indices, but referenced by index templates and used when determining the final configuration of an index.
```